<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnexion</class>
 <widget class="QDialog" name="DialogConnexion">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelConnexion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonSeConnecter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>107</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonSInscrire">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>S'inscrire</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>326</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelPseudo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Pseudo : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="valPseudo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>362</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="formAlignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelMdp">
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="valMdp"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
